src/java.base/share/classes/sun/security/util/Debug.java

```diff
@@ -102,6 +119,11 @@ public static void Help()
         System.err.println("failure       before throwing exception, dump stack");
         System.err.println("              and domain that didn't have permission");
         System.err.println();
+        System.err.println("+timestamp can be appended to any of above options to print");
+        System.err.println("              a timestamp for that debug option");
+        System.err.println("+thread can be appended to any of above options to print");
+        System.err.println("              thread information for that debug option");
+        System.err.println();
```

Comment:
Would it be more reasonable to place these lines after line 113 ("x509") which are the main options?